drivers/include/net/netdev.h

```diff
@@ -70,15 +70,16 @@ enum {
  *          upper layer
  */
 typedef enum {
-    NETDEV_EVENT_ISR,               /**< driver needs it's ISR handled */
-    NETDEV_EVENT_RX_STARTED,        /**< started to receive a packet */
-    NETDEV_EVENT_RX_COMPLETE,       /**< finished receiving a packet */
-    NETDEV_EVENT_TX_STARTED,        /**< started to transfer a packet */
-    NETDEV_EVENT_TX_COMPLETE,       /**< finished transferring packet */
-    NETDEV_EVENT_TX_NOACK,          /**< ACK requested but not received */
-    NETDEV_EVENT_TX_MEDIUM_BUSY,    /**< couldn't transfer packet */
-    NETDEV_EVENT_LINK_UP,           /**< link established */
-    NETDEV_EVENT_LINK_DOWN,         /**< link gone */
+    NETDEV_EVENT_ISR,                       /**< driver needs it's ISR handled */
+    NETDEV_EVENT_RX_STARTED,                /**< started to receive a packet */
+    NETDEV_EVENT_RX_COMPLETE,               /**< finished receiving a packet */
+    NETDEV_EVENT_TX_STARTED,                /**< started to transfer a packet */
+    NETDEV_EVENT_TX_COMPLETE,               /**< finished transferring packet */
+    NETDEV_EVENT_TX_COMPLETE_DATA_PENDING,  /**< finished transferring packet and has data pending flag */
```

Comment:
Is this line still in line with the coding conventions (max 80, 100 if not otherwise possible)